readme.md

```diff
@@ -62,20 +62,22 @@ possible features:
 
 ![](./doc/fixup_roadmap.png)
 
-For updates on the progress see the roadmap progress [thread](https://github.com/nim-works/nimskull/discussions/142?sort=new)
+For updates on the progress refer to the [milestones](https://github.com/nim-works/nimskull/milestones).
+The current ones are used as a way to track progress on a topic, rather than
+for representing fixed milestones. The old, out-of-date thread about roadmap
```

Comment:
If it is old and out of date, probably does not make sense to link it on the readme.

Reply:
My thinking was that it could still be of interest to people seeking additional background and history, but I'm not against removing the mention.